e2e/browser/testApp/components/appContent/index.jsx

```diff
@@ -19,138 +19,101 @@
 // SOFTWARE OR THE USE OR OTHER DEALINGS IN THE SOFTWARE.
 //
 
-import { useSession } from "@inrupt/solid-ui-react";
-import { useState } from "react";
 import {
-  approveAccessRequest,
-  revokeAccessGrant,
-} from "@inrupt/solid-client-access-grants";
-import {
-  getPodUrlAll,
-  saveFileInContainer,
-  getSourceUrl,
-  deleteFile,
-} from "@inrupt/solid-client";
+  login,
+  logout,
+  handleIncomingRedirect,
+  getDefaultSession
+} from "@inrupt/solid-client-authn-browser";
+import { useState, useEffect } from "react";
+import AccessGrants from "../accessGrants"
 
 // This is the content of the file uploaded manually at SHARED_FILE_IRI.
-const SHARED_FILE_CONTENT = "Some content.\n";
+const DEFAULT_ISSUER = "https://login.inrupt.com/";
+const REDIRECT_URL = window.location.href;
+const APP_NAME = "Access Grants browser-based tests app";
+const AccessGrantContainer = ({
+  sessionInfo
+}) => {
+  if (sessionInfo?.isLoggedIn) {
+    return <AccessGrants />;
+  } else {
+    return <></>;
+  }
+}
 
 export default function Home() {
-  const { session } = useSession();
-  const [accessGrant, setAccessGrant] = useState(undefined);
-  const [sharedResourceIri, setSharedResourceIri] = useState(undefined);
+  const [sessionInfo, setSessionInfo] = useState();
+  const [issuer, setIssuer] = useState(DEFAULT_ISSUER);
 
-  const handleCreate = (e) => {
-    // This prevents the default behaviour of the button, i.e. to resubmit, which reloads the page.
-    e.preventDefault();
-    if (typeof sharedResourceIri === "string") {
-      // If a resource already exist, do nothing
-      return;
-    }
-    (async () => {
-      // Create a file in the resource owner's Pod
-      const resourceOwnerPodAll = await getPodUrlAll(session.info.webId);
-      if (resourceOwnerPodAll.length === 0) {
-        throw new Error(
-          "The Resource Owner WebID Profile is missing a link to at least one Pod root."
-        );
-      }
-      const savedFile = await saveFileInContainer(
-        resourceOwnerPodAll[0],
-        Buffer.from(SHARED_FILE_CONTENT),
-        {
-          // The session ID is a random string, used here as a unique slug.
-          slug: `${session.info.sessionId}.txt`,
-          fetch: session.fetch,
-        }
-      );
-      setSharedResourceIri(getSourceUrl(savedFile));
-    })();
-  };
+  const session = getDefaultSession();
```

Comment:
you can lift this up, as it's a singleton, React need not manage this.

Reply:
It's actually not even used, removed it altogether :)

Reply:
d7a5432c8eb6e8d030d44dbddaba1180e5c375b3